src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx

```diff
@@ -6,16 +6,59 @@
 import { ImplementedChange } from 'shared';
 import { datePipe, emDashPipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
 import { routes } from '../../utils/routes';
-import { Link, ListItem, List, Tooltip, Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Link, ListItem, List, Tooltip, Typography, TooltipProps, tooltipClasses } from '@mui/material';
 import PageBlock from '../../layouts/PageBlock';
 import { Link as RouterLink } from 'react-router-dom';
+import { useLayoutEffect, useState } from 'react';
 
 interface ImplementedChangesListProps {
   changes: ImplementedChange[];
   overallDateImplemented?: Date;
 }
 
 const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes, overallDateImplemented }) => {
+  const [bodyWidth, setBodyWidth] = useState<number>(window.document.body.offsetWidth);
+
+  window.document.body.addEventListener('resize', () => {
+    setBodyWidth(window.document.body.offsetWidth);
+  });
+
+  function useWindowSize() {
```

Comment:
lets minimize code duplication by breaking this into a new custom hook in a separate file